src/main/java/baritone/pathing/movement/CalculationContext.java

```
@@ -63,6 +63,9 @@ public class CalculationContext {
     public final List<Block> allowBreakAnyway;
     public final boolean allowParkour;
     public final boolean allowParkourPlace;
+    @Deprecated
+    @JavaOnly
+    public final boolean allowJumpAt256;
```

`CalculationContext` doesn't need a dummy field. The public API is basically "everything in `baritone.api` and nothing else".

---

(That's what I was trying to clarify by using "setting field" instead of "field", guess I failed)